inspirehep/modules/theme/templates/inspirehep_theme/format/record/Base_HTML_brief.html

```diff
@@ -0,0 +1,56 @@
+{#
```

Comment:
Note that the brief formats are now using purely AngularJS for display so they can be removed from the PR.